assets/collage.css

```diff
@@ -112,29 +112,29 @@
 
 .collage-card {
```

Comment:
UX has probably given this more thought than me, but I kind of feel like collage cards should just be their own thing, either via global "collage card" settings or directly in the collage sections settings.

This current approach more or less makes it so either the merchant can't use collage sections OR can't reasonably make use of independent card type settings without making things ugly. Maybe with the occasional exception for the intentionally eclectic aesthetic. Perhaps that's a reasonable choice, but it seems ideal that we wouldn't force that choice on them.

I apologize if this has been addressed and I'm just missing some context.

Reply:
Indeed they can always choose to align the styles so collage cards don't visually clash, and that aligns with our stance of "they always have options to make things not ugly." But that's where I see it being an either-or sort of option for the merchant. You can't feasibly use a collage (with product and collection) AND use different card styles. Sounds like we're ok with this though, I just wanted to present potential options to allow both of this configurations at the same time.